website/docs/r/flow_log.html.markdown

```diff
@@ -100,6 +100,8 @@ The following arguments are supported:
 * `log_group_name` - (Optional) *Deprecated:* Use `log_destination` instead. The name of the CloudWatch log group.
 * `subnet_id` - (Optional) Subnet ID to attach to
 * `vpc_id` - (Optional) VPC ID to attach to
+* `log_format` - The fields to include in the flow log record, in the order in which they
```

Comment:
Looks like the formatting is a little off here, can you please remove the new line and `//`. Its also missing `(Optional)` like the other arguments.

```suggestion
* `log_format` - (Optional) The fields to include in the flow log record, in the order in which they should appear. For a list of available fields, see [Flow Log Records](https://docs.aws.amazon.com/vpc/latest/userguide/flow-logs.html#flow-log-records).
```

along with removing line 104